charmed_openstack_info/data/lp-builder-config/openstack.yaml

```diff
@@ -997,6 +997,11 @@ projects:
     launchpad: charm-cinder-lvm
     repository: https://opendev.org/openstack/charm-cinder-lvm.git
 
+  - name: OpenStack Cinder NFS Charm
+    charmhub: cinder-nfs
+    launchpad: charm-cinder-nfs
+    repository: https://opendev.org/openstack/charm-cinder-nfs.git
+
```

Comment:
Just to note that this is probably wrong, in that it doesn't include a `branches` override.  It's unlikely that this charm needs to go back to bionic-train.  Please select appropriate branches for this charm.

Reply:
I've added branch override